doc/Extensions/Dispatcher-Service.md

```diff
@@ -232,7 +232,7 @@ available, but the testing package is working fine, you can grab it on
 package may be updated and have a new version number).
 
 ```
-apt install python3 python-mysqldb
+apt install python3 python-mysqldb python3-pymysql
```

Comment:
when you change it like this, is python-mysqldb still needed?

Reply:
Yes it is. If you don't have you have an python error

Can not find mysqld module or something like that